index.test-d.ts

```diff
@@ -12,3 +12,14 @@ expectType<IterableIterator<Error>>(aggregateError[Symbol.iterator]());
 for (const error of aggregateError) {
 	expectType<Error>(error);
 }
+
+interface CustomError extends Error {
+	foo: string;
+}
+const customAggregateError = new AggregateError<CustomError>([
+	new Error('foo')
```

Comment:
Can you make an actual Error subclass and use it here instead?

Reply:
like this 6d23959?